docs/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -137,7 +172,28 @@ spec:
 ```
 
 ### Creating Service by specifying a floating IP
-TBD
+If you already have a floating network and want to specify a floating ip for loadbalance service.
```

Comment:
Suggestion: If you want the Service to be exposed to the public internet, ...

Reply:
done